hnn_core/extracellular.py

```diff
@@ -0,0 +1,601 @@
+"""
+Handler classes to calculate extracellular electric potentials, such as the
+Local Field Potential (LFP), at ideal (point-like) electrodes based on net
+transmembrane currents of all neurons in the network.
+
+The code is inspired by [1], but important modifications were made to comply
+with the original derivation [2] of the 'line source approximation method'.
+
+References
+----------
+1 . Parasuram H, Nair B, D'Angelo E, Hines M, Naldi G, Diwakar S (2016)
+Computational Modeling of Single Neuron Extracellular Electric Potentials and
+Network Local Field Potentials using LFPsim. Front Comput Neurosci 10:65.
+2. Holt, G. R. (1998) A critical reexamination of some assumptions and
+   implications of cable theory in neurobiology. CalTech, PhD Thesis.
+"""
+
+# Authors: Mainak Jas <mjas@mgh.harvard.edu>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+#          Sam Neymotin <samnemo@gmail.com>
+#          Christopher Bailey <cjb@cfin.au.dk>
+
+from neuron import h
+
+from copy import deepcopy
+import numpy as np
+from .externals.mne import _validate_type, _check_option
+from numpy.linalg import norm
```

Comment:
imports should be in order of more general to specific

first copy, then numpy, then neuron, then .externals.mne